sdk/networkfunction/Azure.ResourceManager.NetworkFunction/api/Azure.ResourceManager.NetworkFunction.netstandard2.0.cs

```diff
@@ -0,0 +1,233 @@
+        public override string ToString() { throw null; }
+    }
+    public partial class EmissionPoliciesPropertiesFormat
+    {
+        public EmissionPoliciesPropertiesFormat() { }
+        public System.Collections.Generic.IList<Azure.ResourceManager.NetworkFunction.Models.EmissionPolicyDestination> EmissionDestinations { get { throw null; } }
+        public Azure.ResourceManager.NetworkFunction.Models.EmissionType? EmissionType { get { throw null; } set { } }
+    }
+    public partial class EmissionPolicyDestination
+    {
+        public EmissionPolicyDestination() { }
+        public Azure.ResourceManager.NetworkFunction.Models.EmissionDestinationType? DestinationType { get { throw null; } set { } }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct EmissionType : System.IEquatable<Azure.ResourceManager.NetworkFunction.Models.EmissionType>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public EmissionType(string value) { throw null; }
+        public static Azure.ResourceManager.NetworkFunction.Models.EmissionType Ipfix { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.NetworkFunction.Models.EmissionType other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.NetworkFunction.Models.EmissionType left, Azure.ResourceManager.NetworkFunction.Models.EmissionType right) { throw null; }
+        public static implicit operator Azure.ResourceManager.NetworkFunction.Models.EmissionType (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.NetworkFunction.Models.EmissionType left, Azure.ResourceManager.NetworkFunction.Models.EmissionType right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class IngestionPolicyPropertiesFormat
+    {
+        public IngestionPolicyPropertiesFormat() { }
+        public System.Collections.Generic.IList<Azure.ResourceManager.NetworkFunction.Models.IngestionSourcesPropertiesFormat> IngestionSources { get { throw null; } }
+        public Azure.ResourceManager.NetworkFunction.Models.IngestionType? IngestionType { get { throw null; } set { } }
+    }
+    public partial class IngestionSourcesPropertiesFormat
+    {
+        public IngestionSourcesPropertiesFormat() { }
+        public string ResourceId { get { throw null; } set { } }
```

Comment:
Is it a ResourceIdentifier?

Reply:
It's `ResourceIdentifier` when `SourceType` is Resource, but other values for other types.